common-model/src/test/java/com/sequenceiq/common/api/type/TunnelTest.java

```diff
@@ -0,0 +1,13 @@
+package com.sequenceiq.common.api.type;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class TunnelTest {
```

Comment:
Thanks for adding this class. I know it is outside the scope of the changes here, but perhaps it would be useful to cover the rest of {{Tunnel}} as well with some test cases.

Reply:
fine